### ISSUE TITLE
Photometric reconstruction loss mentioned in your conclusion

### ISSUE THREAD

**Issue by tareeqav** [cogaplex-bts/bts#23]
Hello,

Many thanks for your great work! It's very impressive.  Please forgive my ignorance since I am still new to this research.

In your conclusion you mention adopting a photometric reprojection loss.  I assume similar to [Goddard et al.](https://arxiv.org/pdf/1806.01260.pdf) and [Zhou et al.  ](https://people.eecs.berkeley.edu/~tinghuiz/projects/SfMLearner/cvpr17_sfm_final.pdf)

Is my assumption correct?  

I was wondering if you have already begun working on this? What kind of timeline are you expecting?

If you have not yet begun this, I am also considering adding it to your pipeline as the loss function in place of supervised learning.

My Python/C++ skills are pretty good, but my CNN skills still need a lot of work.  Would it be OK to reach out to someone on your team for collaboration on this?  Is that even a possibility?

Thanks,
Sameh

**Comment by cogaplex-bts** [comment]
Yes, you are correct.
We are currently working on a forked project which uses a hybrid training loss defined in terms of photometric reconstruction error and direct supervision.
With this strategy, we can train our bts using both types of dataset: stereo pairs and images with gt.
Thanks for your interest in our work but sorry for that we have no room to give you ragarding this project.
However, you still have a chance to go on your own project if you have a new idea.
Cheers!

Best wishes,
Jin Han Lee

**Comment by a-jahani** [comment]
@tareeqav If you are interested, these two repos might give you an idea and help you with the hybrid loss:
1. [semodepth](https://github.com/Yevkuzn/semodepth) inference is public
2. [semidepth](https://github.com/a-jahani/semiDepth) inference and training are public, and the code is based on monodepth v1.0 code(Goddard et al. you mentioned).

**Comment by tareeqav** [comment]
@a-jahani many thanks! looks very promising
